login.html

```diff
@@ -3,9 +3,49 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Login Page</title>
+    <link rel="stylesheet" href="/css/common/auth.css">
+    <link rel="stylesheet" href="/css/common/reset.css">
+    <link rel="stylesheet" href="css/pages/login.css">
+    <title>판다마켓 - 로그인</title>
 </head>
 <body>
-    
+    <main>
+        <a href="/">
+            <img class="logo" src="/assets/logo/logo_1.png" alt="판다마켓 로고">
+        </a>
+        <div class="input_info">
+            <form>
+                <div class="input_group">
+                    이메일<br>
+                    <input type="email" name="email">
+                </div>
+                <div class="input_group">
+                    비밀번호<br>
+                    <div class="password">
+                        <input id="pwd" type="password" name="password">
+                        <img class="pwd_icon" src="/assets/icons/hide-password.png" alt="비밀번호가리기">
+                    </div>
+                </div>
+                <div class="submit_btn">
+                    <button type="submit">로그인</button>
+                </div>
+            </form>
+            <div class="sign_easy">
+                간편 로그인하기
+                <div>
+                    <a href="https://www.google.com/">
+                        <img class="icon" src="/assets/icons/GoogleBg.png" alt="간편로그인_구글">
```

Comment:
💊 제안
alt 태그에는 공백사용하셔도 됩니다~

```suggestion
                        <img class="icon" src="/assets/icons/GoogleBg.png" alt="간편로그인 구글">
```